tests/wicked/aggregate/sut/t21_teaming_ifcfg_check_port_config.pm

```diff
@@ -0,0 +1,81 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: wicked teamd
+# Summary: Teaming, ifcfg - check port config
+# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>
+#             Jose Lausuch <jalausuch@suse.com>
+#             Clemens Famulla-Conrad <cfamullaconrad@suse.de>
+
+use Mojo::Base 'wickedbase';
+use Mojo::JSON qw(decode_json);
+use Mojo::JSON::Pointer;
+use testapi;
+
+sub validate_team_port_config {
+    my ($self, $ctx) = @_;
+    my $iface1 = $ctx->iface();
+    my $iface2 = $ctx->iface2();
+
+    $self->wicked_command('ifup', 'team0');
+    $self->validate_interfaces('team0', $ctx->iface(), $ctx->iface2(), 0);
+    $self->ping_with_timeout(type => 'host', interface => 'team0', count_success => 30, timeout => 4);
+
+    my $jpointer = Mojo::JSON::Pointer->new(decode_json(script_output('teamdctl team0 config dump actual')));
```

Comment:
this line requires some explanation ...

Reply:
In code or here?
`teamdctl` can be usesed to see the teamd configuration of some interfaces. And this what I'm doing here. 
The output is JSON, so I parse the output as JSON and pass it to JSON::Pointer. The JSON::Pointer is a helper to search in json like you can do with `dom` objects. We need this for easy evaluation of the output afterwards.

The output is:
```json
{
    "device": "team0",
    "mcast_rejoin": {
        "count": 1
    },
    "notify_peers": {
        "count": 1
    },
    "ports": {
        "ens4": {
            "prio": 10
        },
        "ens5": {
            "prio": 1,
            "sticky": true
        }
    },
    "runner": {
        "hwaddr_policy": "same_all",
        "name": "activebackup"
    }
}
```
and we will check if the `ports' config has `prio` and `sticky` set if it should.